api/models/User.js

```diff
@@ -221,7 +229,31 @@ module.exports = bookshelf.Model.extend(merge({
     DELETE FROM posts_tags WHERE post_id in (select id from posts WHERE user_id = ${this.id});
     DELETE FROM groups_posts WHERE post_id in (select id from posts WHERE user_id = ${this.id});
     DELETE FROM votes WHERE user_id = ${this.id};
-    DELETE FROM users WHERE id = ${this.id};
+
+    UPDATE users SET 
+    active = false, 
+    settings = NULL, 
+    name = 'Deleted User', 
+    avatar_url = NULL, 
+    bio = NULL, 
+    banner_url = NULL,
+    location = NULL,
+    url = NULL,
+    tagline = NULL,
+    stripe_account_id = NULL,
+    location_id = NULL,
+    contact_email = NULL,
+    contact_phone = NULL,
+    email = '${uuid.v4()}@hylo.com',
```

Comment:
still thinking about this one too... I guess this looks best

Reply:
Yeah, just needed a unique value